Assets/Scripts/PlayerControl.js

```diff
@@ -1,43 +1,46 @@
 ﻿#pragma strict
 
 public var speed : float = 3; // nopeus, voi muokata editorissa
-public var cameraPrefab : Camera; // kamera - prefab
 
 private var rigidBody : Rigidbody2D; // rigidbody2d
 private var player : Player;
-private var cameraObj : Camera;
 
 
 function Start() {
 
   rigidBody = GetComponent(Rigidbody2D);
   player = GetComponent(Player);
-  cameraObj = Instantiate(cameraPrefab);
 
 }
 
 function FixedUpdate() {
-  // look towards mouse pointer
-  var mousePos : Vector2 = cameraObj.ScreenToWorldPoint(Input.mousePosition);
-  var heading : Vector2 = (mousePos - transform.position).normalized;
-  transform.up = heading;
-
-  if (Input.GetAxis("Vertical")) {
-    // move if buttons are pressed
-    var inputDir : float = Input.GetAxis("Vertical");
-    var velocity : Vector2 = Vector2(inputDir * heading.x, inputDir * heading.y);
-    rigidBody.velocity = Vector2.ClampMagnitude(velocity * speed, speed);
-
-  } else {
-    // Don't move when buttons are not pressed
-    rigidBody.velocity = Vector2.zero;
-  }
 
-  cameraObj.transform.position.x = transform.position.x;
-  cameraObj.transform.position.y = transform.position.y;
+  try {
```

Comment:
Mihin tarvii try-catchia? Jos täällä tulee virhe, joka catchataan, ei pelaaja toimi ja käyttäjä ei tiedä mitään. Ja tässä tulevat virheet pitäis johtua ainoastaan siitä että pelaajan prefabbi on rikki tai koodissa on muuten vaan vikaa.

Jos pakko käyttää try-catchia niin sitten vain sen kohdan ympärillä mistä se mahdollinen virhe tulee.